Svc/CmdDispatcher/CommandDispatcherImpl.hpp

```diff
@@ -72,7 +72,7 @@ namespace Svc {
             //!
             //!  \param portNum the number of the incoming port.
             //!  \param data the buffer containing the command.
-            //!  \param context a user value returned with the statuss
+            //!  \param context a user value returned with the status
```

Comment:
I always worry about changes like this, maybe `statuses`?

Reply:
this is correct